### PR TITLE
Adding useragent to msdeploy exe

### PR DESCRIPTION
**Pull request by pragupta-ms**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1 [hunk 1]
```diff
@@ -135,6 +135,31 @@ function Get-MsDeployCmdArgs
     if( -not [String]::IsNullOrEmpty($AdditionalArguments)){
         $msDeployCmdArgs += ( " " + $AdditionalArguments)
     }
+	
+    $collectionUri = Get-VstsTaskVariable -Name System.TeamFoundationCollectionUri -Require
```
Comment (ericsciple) [inline Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1]: This will error with Command not found

Comment (pragupta-ms) [inline Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1]: resolved

File: Tasks/AzureRmWebAppDeployment/Utility.ps1 [hunk 2]
```diff
@@ -136,6 +136,31 @@ function Get-MsDeployCmdArgs
         $msDeployCmdArgs += ( " " + $AdditionalArguments)
     }
 
+    $collectionUri = Get-VstsTaskVariable -Name System.TeamFoundationCollectionUri -Require
```
Comment (Ajay-MS) [inline Tasks/AzureRmWebAppDeployment/Utility.ps1]: @pragupta-ms - Can we extract out this to a method which will return $userAgent , and in this method just append $userAgent to $msDeployCmdArgs

Comment (pragupta-ms) [inline Tasks/AzureRmWebAppDeployment/Utility.ps1]: resolved

File: Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1 [hunk 3]
```diff
@@ -185,4 +190,30 @@ function Run-MsDeployCommand
     Write-Host (Get-LocalizedString -Key "Running msdeploy command: `n`t{0}" -ArgumentList $msDeployCmdForLogs)
     Run-Command -command $msDeployCmd
     Write-Host (Get-LocalizedString -Key "msdeploy command ran successfully.")
+}
+
+function Get-UserAgentString
```
Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1]: Is this function going to be reused across multiple Azure tasks? If so, should it reside in common?

Comment (pragupta-ms) [inline Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1]: Will move it there if the use cases increase

**Pull request merged**